client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -265,3 +265,161 @@ h2.action-panel__title {
 	display:flex;
 	flex-direction: column;
 }
+
+
+
+.form-text-input-with-affixes__suffix {
+	font-weight: normal;
+}
```

Comment:
This selector needs to be nested inside another selector, otherwise it will apply to all instances of this control.